attribution-graphs-frontend/attribution_graph/init-cg.js

```
@@ -0,0 +1,206 @@
+    isShowAllLinks: '',
+    isSyncEnabled: '',
+    subgraph: null,
+    isEditMode: 1,
+    isHideLayer: data.metadata.scan == util.scanSlugToName.h35 || data.metadata.scan == util.scanSlugToName.moc,
+    sg_pos: '',
+    isModal: true,
+    isGridsnap,
+    ...data.qParams
+  }
+
+  if (visState.clickedId?.includes('supernode')) delete visState.clickedId
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)